CONTRIBUTING.md

```diff
@@ -1,3 +1,20 @@
 # Contributing
 
 We follow the [IPython Contributing Guide](https://github.com/ipython/ipython/blob/master/CONTRIBUTING.md).
+
+## Release guidelines
+
+When there is a release, update the appropriate schema file to reflect the new
+version (both in its top-level description and in its minimum minor/major
+version numbers in the spec) and copy the file to an archived schema for that
+particular version. Also, update the minor format for tests.
```

Comment:
Add language:

> *If* the spec has material changes, ...

And this perhaps more appropriately happens at the time of a pull-request introducing such a change, rather than release-time.

Reply:
Doing this at pull-request time will also have the result of correct behavior for folks working from master, rather than incorrect behavior until a release is made.

Reply:
Having it at release time means that we can combine changes happening between releases into a single spec upgrade, though. Otherwise the spec could jump between, say, 5.0 to 5.5 in one package release.